src/core/server/http/router/request.ts

```diff
@@ -37,6 +37,7 @@ export interface KibanaRequestRoute {
   options: Required<RouteConfigOptions>;
 }
 
+const securedHeaders = ['authorization'];
```

Comment:
authHeaders is a map and we don't know its keys. @elastic/kibana-security we can make this value configurable when needed.

Reply:
__question:__ `secured` can give a false sense of security especially when it's used in OSS or when security plugin is disabled, just sayin' :slightly_smiling_face:  What does `secured` mean here?

Reply:
Headers aren't exposed by `getFilteredHeaders` method. We can call it `filteredHeaders` but it doesn't show an intention and doesn't answer why. 🤔

Reply:
How about `secretHeaders`

Reply:
works for me